### ISSUE TITLE
Licenses of dependencies

### ISSUE THREAD

**Issue by frankdiox** [tarlepp/angular-sailsjs-boilerplate#67]
Thank you very much for this boilerplate, @tarlepp , quite useful :)

I've seen you have included Highcharts as a dependency lately and I also need some charts for my project. However, looks like Highcharts license is not completely free (only for non-commercial), so I should not use it. This makes me wonder whether there is any other dependency license with the same issue in your boilerplate. As a suggestion, perhaps it would be nice to have a list of this kind of dependencies in the readme.md.

Thanks again!

**Comment by tarlepp** [comment]
Yeah, this is something that should be mentioned. Feel free to make PR about this.

**Comment by tarlepp** [comment]
I think that list of 3rd party libraries is enough for now or just link to ```bower.json``` and ```package.json```. Basically I don't want to update list of 3rd licenses...
